templates/zerver/config_error.html

```diff
@@ -53,6 +53,9 @@ <h1 class="lead">Configuration error</h1>
                         {{ render_markdown_path('zerver/'+social_backend_name+'-error.md',
                         {"root_domain_uri": root_domain_uri, "settings_path": secrets_path, "secrets_path": secrets_path,
                         "client_id_key_name": "social_auth_" + social_backend_name + "_key"}) }}
+                        <p>
+                            <b>Note: </b>Configuring auth backend is <b>optional</b> and is required only if you are working on an issue related to auth backend.
+                        </p>
```

Comment:
I don't think this note is necessary if we address the other text clearly.

Reply:
And more importantly, I believe this text is shared with production servers, so the reference to "working on an issue" isn't correct.